app/views/govuk_web_banners/_global_banner.html.erb

```diff
@@ -0,0 +1,11 @@
+<% global_banner = GovukWebBanners::GlobalBanner.for_path(request.path) %>
+<% if global_banner.present? %>
+  <%= render "govuk_publishing_components/components/global_banner", {
+      title: global_banner.items.first.title,
+      title_path: global_banner.items.first.title_href,
+      text: global_banner.items.first.text,
+      show_arrows: global_banner.show_arrows,
```

Comment:
Is this a valid option on the component? It's not in the guide: https://components.publishing.service.gov.uk/component-guide/global_banner

Reply:
Removed (see above for discussion)